components/nav.js

```diff
@@ -71,35 +77,27 @@ export function TabNav() {
 // }
 
 function RecipesScreen() {
-  return (
-    <View style={styles.view}>
-      <Recipes></Recipes>
-    </View>
-  );
+  return <RecipesTab style={styles.view}></RecipesTab>;
 }
 
 function SavedScreen() {
-  return ( 
-    <View style={styles.view}>
-      <Text style={styles.text}>Settings!</Text>
-    </View >
-  ); 
+  return <OneRecipe style={styles.view}></OneRecipe>;
 }
 
 const styles = StyleSheet.create({
   view: {
-    ...view
+    ...view,
   },
   navContainer: {
-    backgroundColor: 'white',
-    paddingTop: 5,
-    paddingBottom: 5
+    backgroundColor: "white",
+    paddingTop: 10,
+    paddingBottom: 10,
   },
   container: {
     flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
-    padding: 24
-  }
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundColor: "#F5FCFF",
```

Comment:
Also should be a colour from the global stylesheet. We should stay away from hard-coding colours because it makes everything harder in the future.